raiden/tests/unit/rif_comms/test_rif_comms_client.py

```diff
@@ -159,3 +153,36 @@ def test_send_lumino_message(self):
                 message=Msg(payload=str.encode(json.dumps(some_raiden_message)))
             )
         )
+
+    @pytest.mark.skip(reason="ignore")
+    def test_two_clients_topic_subscription(self):
+        notification_stream_1 = self.rif_comms_client1.connect()
+
+        notification_stream_2 = self.rif_comms_client2.connect()
+
+        one_own_sub = self.rif_comms_client1.subscribe_to(LUMINO_1_ADDRESS)
+        one_two_sub = self.rif_comms_client1.subscribe_to(LUMINO_2_ADDRESS)
+
+        two_own_sub = self.rif_comms_client2.subscribe_to(LUMINO_2_ADDRESS)
+        two_one_sub = self.rif_comms_client2.subscribe_to(LUMINO_1_ADDRESS)
+
+        peer_id_1 = self.rif_comms_client2._get_peer_id(LUMINO_2_ADDRESS)
```

Comment:
this seems unfinished; maybe we should replace this `_get_peer_id` call (once the code no longer hangs without it) and check for both subscriptions on each peer instead

Reply:
yep, this test was for checking that two nodes can subscribe to each other topic and receive messages, but we need to iterate over this

right now the test that ensures this works is to run two lumino nodes and send messages between each other, but is not something automatic

Reply:
resolved through #166